.github/workflows/unit_test_bcc.yml

```diff
@@ -9,4 +9,4 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: test bcc in docker container
-      run: podman run -v $(pwd):/home quay.io/sustainable_computing_io/kepler_builder:go1.18 bash -c "cd /home; export PATH=/usr/local/go/bin:$PATH; make test-verbose"
+      run: sudo podman run --privileged -v $(pwd):/home -v /sys:/sys -v /lib/modules:/lib/modules -v /usr/src:/usr/src quay.io/sustainable_computing_io/kepler_builder:go1.18 bash -c "cd /home; export PATH=/usr/local/go/bin:$PATH; make test-verbose"
```

Comment:
may I know why we need 
- privileged, as sudo and privileged
- lib module mount?

Reply:
ebpf needs to run in privileged mode and access /lib/modules to find path to kernel header